src/AddTodoForm.js

```diff
@@ -1,18 +1,29 @@
 import React from 'react';
 
-export default function AddTodoForm(props) {
+export default function AddTodoForm({ onAddTodo }) {
+  const [todoTitle, setTodoTitle] = React.useState('');
+
+  function handleTitleChange(event) {
+    const newTodoTitle = event.target.value;
+    setTodoTitle(newTodoTitle);
+  }
```

Comment:
Something to know/look out for as you see other code floating around is that `event` is a booger just like `props` and devs will commonly shorten it to just `e`. It's also common for simpler handlers like this to forego creating a separate variable for the value and just do everything in the setter function. No need to make any changes here, just exposing you to other methods of doing the same thing.

```suggestion
  function handleTitleChange(e) {
    setTodoTitle(e.target.value);
  }
```